packages/modules/order/src/services/order-module-service.ts

```diff
@@ -3228,11 +3328,12 @@ export default class OrderModuleService
         )
       }
 
+      const initialDiff = MathBN.sub(
+        summary.totals.transaction_total,
+        initialTrxTotal
+      )
       summary.totals.pending_difference = new BigNumber(
-        MathBN.sub(
-          summary.totals.current_order_total,
-          summary.totals.transaction_total
-        )
+        MathBN.sub(summary.totals.pending_difference, initialDiff)
```

Comment:
The previous diff looks correct to me here - if `pending_difference` is how much a customer has to pay, shouldn't it be the current total subtracted by the current transaction total?

Reply:
Return requests create a negative pending value, if we just get the current total minus the transactions we overwrite them.

Reply:
Ah I see, that makes sense. The confusion here then is more about why current_order_total includes the cost of the returned items then. I don't know if there's anything to fix here, just some education is probably needed as comments inline 🤞🏻

Reply:
we can discuss this later once you're back, merging this in for now 🤞🏻